src/Compilers/Core/Portable/Diagnostic/DiagnosticDescriptor.cs

```diff
@@ -227,6 +227,8 @@ private static ReportDiagnostic MapSeverityToReport(DiagnosticSeverity severity)
                     return ReportDiagnostic.Warn;
                 case DiagnosticSeverity.Error:
                     return ReportDiagnostic.Error;
+                case DiagnosticSeverity.Suggestion:
```

Comment:
Sorry, I'm not sure I understand. Suggestion is not just about visual presentation. It is mutually exclusive from Error and Warning and can affect the user's build. Given the bulk of diagnostics that will be created in the suggestion category, the design team felt that there was enough critical mass to warrant a new severity.

FWIW, if you'd like to be involved in the design discussion around this, you're definitely welcome! However, a code review is probably not right place for this to happen.